content/working_with/console/widgets/plugins.md

```diff
@@ -8,6 +8,7 @@ Displays a list of all the plugins uploaded to the current tenant, according to
 
 The widget displays the following information:
 
+* **Plugin name**
```

Comment:
Possibly we can also add plugin icon here.

Reply:
Done